### DISCUSSION TITLE
Borg backup failed, started from AIO console

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#2083] [Questions] [answered]
Hi there,
I had created an instance of nextcloud AIO today based on a fresh installed linux machine. 
I'm using a yml file mentioned at an other [discussion](https://github.com/nextcloud/all-in-one/discussions/588#discussioncomment-2811152).
Now I would like start the backup on the AIO console, but I got the massege "backup failed" with the log:

```
configuration.json not present. Cannot perform the backup!
/usr/bin/start.sh: line 42: /nextcloud_aio_volumes/nextcloud_aio_mastercontainer/data/backup_archives.list: No such file or directory
chmod: /nextcloud_aio_volumes/nextcloud_aio_mastercontainer/data/backup_archives.list: No such file or directory

```

It seems that the creation of the container points to an other directory in /var/lib/docker/volumes? 

```
brw------- 1 root root  8, 2 27. Feb 20:30 backingFsBlockDev
-rw------- 1 root root 65536 28. Feb 12:45 metadata.db
drwx-----x 3 root root  4096 28. Feb 12:25 nextcloud_aio_apache
drwx-----x 3 root root  4096 28. Feb 12:45 nextcloud_aio_backup_cache
drwx-----x 3 root root  4096 28. Feb 12:23 nextcloud_aio_database
drwx-----x 3 root root  4096 28. Feb 12:23 nextcloud_aio_database_dump
drwx-----x 3 root root  4096 28. Feb 12:45 nextcloud_aio_elasticsearch
drwx-----x 3 root root  4096 28. Feb 12:45 nextcloud_aio_mastercontainer
drwx-----x 3 root root  4096 28. Feb 12:23 nextcloud_aio_nextcloud
drwx-----x 3 root root  4096 28. Feb 12:23 nextcloud_aio_nextcloud_data
drwx-----x 3 root root  4096 28. Feb 12:23 nextcloud_aio_redis
drwx-----x 3 root root  4096 28. Feb 12:18 nextcloud_nextcloud_aio_mastercontainer

```
The subfolder nextcloud_aio_mastercontainer is empty, only nextcloud_nextcloud_aio_mastercontainer contains the configuration.json.

Have you been any suggestions regarding this failure?

Thanks

**Comment by szaimen** [comment, marked as answer]
Hi, can you post the docker run command or docker-compose file here that you used?

**Reply by szaimen** [reply]
https://github.com/nextcloud/all-in-one/pull/2091
